frontend/src/components/PasswordInput.tsx

```diff
@@ -1,9 +1,16 @@
-import { FormControl, FormLabel, Input } from '@chakra-ui/react'
+import { FormControl, FormLabel, Input } from '@chakra-ui/react';
+import { ChangeEvent } from 'react';
 
-const PasswordInput = () =>
+interface Props {
+    value: string;
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Så også lige den her, den burde tage en string ind istedet for et event, da den del kan klares i komponenten.
så
onChange: (event: string) => void;

og nede i on change burde den så hedde

 onChange={(e)=> onChange(e.target.value)} istedet